sentences/en/climate_HassClimateGetTemperature.yaml

```diff
@@ -2,18 +2,19 @@ language: "en"
 intents:
   HassClimateGetTemperature:
     data:
-      # What is...
+      # Get temperature of a climate in the same area as a satellite device
+      - sentences:
+          - "<what_is> [the] [current] <temp> [[in] here]"
```

Comment:
Instead of `[[in] here]`, you should use `[<here>]`, which includes “in here” but also more variations.

Reply:
Done: [9177e6..b887e9](https://github.com/home-assistant/intents/compare/9177e6b793e0cd84f02c16fe43986b47ac04bd42..b887e91e2ddd86c03be98158ac6f327f9a8a1953)